components/01-globals/typography/link/_link.scss

```diff
@@ -5,6 +5,7 @@ $link__font-weight        : $font-weight-normal !default;
 $link__color-hover        : $color-secondary !default;
 $link__line-height        : $font-line-height !default;
 $link__transition         : $transition-base !default;
+$link__text-decoration    : none !default;
```

Comment:
I'd set `underline` as default. at least for text links. What do you think?

Reply:
I agree it should be with underline, but I didn't want to change alpaca styles - I want to change only variable scope - not affecting to the main style. (I need to consult that with @Aekal)

Reply:
ok, so if you need to close this PR, please merge & after consultation you can open new issue on alpaca